From 15c20caca6bffeb0e59cfd0f6c01c7a58f53aa90 Mon Sep 17 00:00:00 2001
From: Lionel Debieve <lionel.debieve@foss.st.com>
Date: Fri, 7 Jan 2022 08:55:25 +0100
Subject: [PATCH 0244/1695] plat-stm32mp2: conf: enable RIFSC driver

Enable the RIFSC driver and add the firewall framework
support for DT probe.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git b6fc71ad281fbee4b4809e1c3c7b5619ab4bc3ec plat-stm32mp2: conf: enable RIFSC driver]
Change-Id: I986656b022927634bb6d3e20f76aa3a8de25ae17
Signed-off-by: Lionel Debieve <lionel.debieve@st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index 7b9a66749..7031b87e4 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -45,6 +45,7 @@ $(call force,CFG_HALT_CORES_ON_PANIC_SGI,15)
 $(call force,CFG_INIT_CNTVOFF,y)
 $(call force,CFG_SECURE_TIME_SOURCE_CNTPCT,y)
 $(call force,CFG_STM32_SHARED_IO,y)
+$(call force,CFG_STM32_FIREWALL,y)
 $(call force,CFG_STM32MP_CLK_CORE,y)
 $(call force,CFG_STM32MP25_CLK,y)
 $(call force,CFG_STM32MP25_RSTCTRL,y)
@@ -65,6 +66,7 @@ CFG_STM32MP_OPP_COUNT ?= 3
 
 CFG_STM32_GPIO ?= y
 CFG_STM32_RIF ?= y
+CFG_STM32_RIFSC ?= y
 CFG_STM32_UART ?= y
 
 # Default enable some test facitilites
-- 
2.47.2

